pkg/security/resolvers/sbom/resolver.go

```diff
@@ -84,7 +85,7 @@ func (s *SBOM) reset() {
 // NewSBOM returns a new empty instance of SBOM
 func NewSBOM(host string, source string, id string, cgroup *cgroupModel.CacheEntry) (*SBOM, error) {
 	return &SBOM{
-		files:          make(map[string]*Package),
+		files:          make(map[uint64]*Package),
```

Comment:
Can you expand a bit on what is the goal with this ? Is it to reduce the size of the files map ?

Reply:
Sure. I added some numbers in the description